content/general/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Data Types'
+Description: 'Data is classified into data types that tell the compiler how the data is intended to be used.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Data Types'
+  - 'Types'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/build-web-apps-with-asp-net'
+---
+
+Data is the foundation of all programs. In programming, data is classified into data types to let the compiler know how the data is being used. 
+
+Data types can be simple or complex. Simple data types are the most basic types and usually hold a single value.  A complex data type can be composed of multiple data types and often group multiple values.
+
+Some simple data types are:
+
+| Data Type | Description |
+| --- | --- |
+| `Number` | Numbers include `-1`, `0`, `1`, and so on. (Depending on the language decimals are a different data type.) |
+| `String` | Strings is just text. Either single quotes (`''`) or double quotes (`""`) can be used.  For example: `"Codecademy is awesome! 🖥️⌨️"` |
+| `Boolean` | Booleans are a `true` or `false` value. Ex: `codeingIsFun = true` |
+| `null` | `null` represents no value. |
+| `Undefined` | `Undefined` also represents no value. |

```

Comment:
```suggestion
| number | Numbers include `-1`, `0`, `1`, and so on. (Depending on the language, numbers with decimals are of a different type.) |
| string | Strings are text. Depending on the language, single quotes (`''`) or double quotes (`""`) can be used.  For example: `"Codecademy is awesome! 🖥️⌨️"` |
| boolean | Booleans hold a true or false value. |
| null | `null` or `nil` represents the absence of a value. |
| undefined | Undefined represents a value that has not been defined. |

```

I like how you added for the numbers that "depending on the language." I'd like to continue with this narrative and stay technically accurate all across the board for this entry.

Reply:
Before this, I didn't know how to make a table using MD. So I just look at the table for JS data types and I saw a lot of spaces so I did that.